src/noteburst/config.py

```diff
@@ -322,6 +328,25 @@ def parsed_worker_token_scopes(self) -> list[str]:
         """
         return [t.strip() for t in self.worker_token_scopes.split(",") if t]
 
+    @property
+    def nublado_image(self) -> NubladoImage:
+        """The JupyterLab image to use for the pod."""
+        match self.image_selector:
+            case JupyterImageSelector.recommended:
+                return NubladoImageByClass(
+                    image_class=NubladoImageClass.RECOMMENDED
+                )
+            case JupyterImageSelector.weekly:
+                return NubladoImageByClass(
+                    image_class=NubladoImageClass.LATEST_WEEKLY
+                )
+            case JupyterImageSelector.reference:
+                return NubladoImageByReference(reference=self.image_reference)
+            case _:
+                raise ValueError(
+                    f"Unknown image selector: {self.image_selector}"
+                )
```

Comment:
You could use [assert_never](https://typing.python.org/en/latest/guides/unreachable.html#assert-never-and-exhaustiveness-checking) here to catch errors at type checking time (like if we add new members to `JupyterImageSelector`) , given that Pydantic will already validate that `image_selector` is a valid `JupyterImageSelector`.